### ISSUE TITLE
Casting doesn't work with Festify

### ISSUE THREAD

**Issue by mweber03** [Festify/app#117]
If I cast my Spotify to a Chromecast enabled device while Festify is running, the songs don't play they just skip through. Spotify shows that one second elapses then it goes to the next song and continues until the list is done, while never actually playing a song.

**Comment by NeoLegends** [comment]
Yes, Spotify Connect playback is not supported yet. But we will come around to implement that eventually. :) This is one of our goals with the new Festify.

**Comment by mweber03** [comment]
Any updates on this?

**Comment by leolabs** [comment]
Not yet, but it's still on our list. We're currently rewriting the UI to make it more stable and performant as we've gotten many complaints about that. When that's finished, we may start working on this feature.

**Comment by Quinncuatro** [comment]
Bump. Any word on when this is being added?

**Comment by cyrano007** [comment]
Any updates on this?

**Comment by leolabs** [comment]
Hey @cyrano007,
we're still on that. There's one problem that's holding us back at the moment: With Spotify connect, we don't get any real-time updates as opposed to local playback in the browser. That means we have to poll the player's status every few seconds which could be problematic in regards to Spotify's rate limiting. When we find a solution to that problem, we'll implement it.

**Comment by mstroecker** [comment]
Hey,
>That means we have to poll the player's status every few seconds

I know it is not ideal, but couldn't you just update rarely or not at all? As long as no one changes the players state by hand, you could predict when the next track should be played.

**Comment by leolabs** [comment]
That may work, we'll try that out. Thanks for the pointer :)

**Comment by Quinncuatro** [comment]
@leolabs - make sure to take into account crossfade!
